src/main/java/scm/address/logic/parser/AddScheduleCommandParser.java

```diff
@@ -66,6 +66,18 @@ public AddScheduleCommand parse(String userInput) throws ParseException {
         String endDay = endDateTime.substring(8, 10);
         String endHour = endDateTime.substring(11, 13);
         String endMinute = endDateTime.substring(14, 16);
+        String start = startYear + "-" + startMonth + "-" + startDay + "T" + startHour + ":" + startMinute;
+        String end = endYear + "-" + endMonth + "-" + endDay + "T" + endHour + ":" + endMinute;
+
+        try {
+            LocalDateTime dateTime1 = LocalDateTime.of(Integer.parseInt(startYear), Integer.parseInt(startMonth),
+                    Integer.parseInt(startDay), Integer.parseInt(startHour), Integer.parseInt(startMinute));
+            LocalDateTime dateTime2 = LocalDateTime.of(Integer.parseInt(endYear), Integer.parseInt(endMonth),
+                    Integer.parseInt(endDay), Integer.parseInt(endHour), Integer.parseInt(endMinute));
+        } catch (DateTimeException e) {
+            throw new ParseException("Invalid date and/or time.", e);
+        }
+
```

Comment:
It would be better if instead of taking substrings, LocalDateTime.parse(string, FORMATTER) would be used instead, so that invalid date inputs would be caught.

Reply:
Thanks just fixed it